ansible_base/lib/workload_identity/workload_identity_tokens.py

```
@@ -25,6 +34,18 @@ class WorkloadIdentityTokenRequestSerializer(serializers.Serializer):
         allow_empty=False,
         help_text=_("Workload details to include in the JWT as claims."),
     )
+    workload_ttl_seconds = serializers.IntegerField(
+        required=False,
+        allow_null=True,
+        min_value=1,
+        max_value=WORKLOAD_TTL_MAX_SECONDS,
+        help_text=_(
+            f"Optional workload-specific TTL override in seconds (1–{WORKLOAD_TTL_MAX_SECONDS}). "
```

Ruff (0.15.2)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)